webhook/aws/sns_handler_test.go

```diff
@@ -52,19 +47,17 @@ const (
   } }`
 )
 
-func SetUpTestSNSServer() (*SNSServer, *mockSVC, *mux.Router)  {
+func SetUpTestSNSServer() (*SNSServer, *MockSVC, *mux.Router)  {
 	
 	v := SetUpTestViperInstance(TEST_AWS_CONFIG)
 	
-	awsCfg, err := NewAWSConfig(v.Sub(AWSKey))
-	m := &mockSVC{}
+	awsCfg, _ := NewAWSConfig(v.Sub(AWSKey))
+	m := &MockSVC{}
 	
 	ss := &SNSServer{
 		Config: *awsCfg,
 		SVC: m,
 	}
-	fmt.Println(ss)
-	fmt.Println(err)
 	
 	selfURL := &url.URL{
 		Scheme:   "http",
```

Comment:
If possible, use `httptest` or similar approach to avoid hardcoding the port.  These tests will break if a client has something running on port 8090.  Best to bind to an unused port for testing.